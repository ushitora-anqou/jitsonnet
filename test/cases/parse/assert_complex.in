assert assert x == 3.0 : "s1"; 0.0 == 0.0 : "s2"; 0.0
