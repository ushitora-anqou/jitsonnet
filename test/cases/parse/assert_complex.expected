{ Syntax.v =
  (Syntax.Assert
     ({ Syntax.v =
        ({ Syntax.v =
           (Syntax.Assert
              ({ Syntax.v =
                 ({ Syntax.v =
                    (Syntax.Binary
                       ({ Syntax.v = (Syntax.Var "x");
                          loc =
                          { Syntax.fname =
                            "test/cases/parse/assert_complex.in";
                            ran =
                            (Some { Syntax.startpos =
                                    { Syntax.line = 1; column = 14 };
                                    endpos = { Syntax.line = 1; column = 15 }
                                    })
                            }
                          },
                        `Equal,
                        { Syntax.v = (Syntax.Number 3.);
                          loc =
                          { Syntax.fname =
                            "test/cases/parse/assert_complex.in";
                            ran =
                            (Some { Syntax.startpos =
                                    { Syntax.line = 1; column = 19 };
                                    endpos = { Syntax.line = 1; column = 22 }
                                    })
                            }
                          }));
                    loc =
                    { Syntax.fname = "test/cases/parse/assert_complex.in";
                      ran =
                      (Some { Syntax.startpos =
                              { Syntax.line = 1; column = 14 };
                              endpos = { Syntax.line = 1; column = 22 } })
                      }
                    },
                  (Some { Syntax.v = (Syntax.String "s1");
                          loc =
                          { Syntax.fname =
                            "test/cases/parse/assert_complex.in";
                            ran =
                            (Some { Syntax.startpos =
                                    { Syntax.line = 1; column = 25 };
                                    endpos = { Syntax.line = 1; column = 29 }
                                    })
                            }
                          }));
                 loc =
                 { Syntax.fname = "test/cases/parse/assert_complex.in";
                   ran =
                   (Some { Syntax.startpos = { Syntax.line = 1; column = 7 };
                           endpos = { Syntax.line = 1; column = 29 } })
                   }
                 },
               { Syntax.v =
                 (Syntax.Binary
                    ({ Syntax.v = (Syntax.Number 0.);
                       loc =
                       { Syntax.fname = "test/cases/parse/assert_complex.in";
                         ran =
                         (Some { Syntax.startpos =
                                 { Syntax.line = 1; column = 31 };
                                 endpos = { Syntax.line = 1; column = 34 } })
                         }
                       },
                     `Equal,
                     { Syntax.v = (Syntax.Number 0.);
                       loc =
                       { Syntax.fname = "test/cases/parse/assert_complex.in";
                         ran =
                         (Some { Syntax.startpos =
                                 { Syntax.line = 1; column = 38 };
                                 endpos = { Syntax.line = 1; column = 41 } })
                         }
                       }));
                 loc =
                 { Syntax.fname = "test/cases/parse/assert_complex.in";
                   ran =
                   (Some { Syntax.startpos = { Syntax.line = 1; column = 31 };
                           endpos = { Syntax.line = 1; column = 41 } })
                   }
                 }));
           loc =
           { Syntax.fname = "test/cases/parse/assert_complex.in";
             ran =
             (Some { Syntax.startpos = { Syntax.line = 1; column = 7 };
                     endpos = { Syntax.line = 1; column = 41 } })
             }
           },
         (Some { Syntax.v = (Syntax.String "s2");
                 loc =
                 { Syntax.fname = "test/cases/parse/assert_complex.in";
                   ran =
                   (Some { Syntax.startpos = { Syntax.line = 1; column = 44 };
                           endpos = { Syntax.line = 1; column = 48 } })
                   }
                 }));
        loc =
        { Syntax.fname = "test/cases/parse/assert_complex.in";
          ran =
          (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
                  endpos = { Syntax.line = 1; column = 48 } })
          }
        },
      { Syntax.v = (Syntax.Number 0.);
        loc =
        { Syntax.fname = "test/cases/parse/assert_complex.in";
          ran =
          (Some { Syntax.startpos = { Syntax.line = 1; column = 50 };
                  endpos = { Syntax.line = 1; column = 53 } })
          }
        }));
  loc =
  { Syntax.fname = "test/cases/parse/assert_complex.in";
    ran =
    (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
            endpos = { Syntax.line = 1; column = 53 } })
    }
  }